jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -286,6 +294,11 @@ YUI.add('deployment-flow', function() {
         // Here we need to just prevent the deployment flow to close.
         return;
       }
+      this.props.analytics.send(
+        'Deployment Flow',
+        'Button click',
+        'Close - Exit deployment'
```

Comment:
We should also be able to see what's it the percentage of usage from the event flow.